revscoring/scorer_models/thresholds.py

```diff
@@ -0,0 +1,167 @@
+from collections import defaultdict
+
+import tabulate
+
+
+class Thresholds(dict):
```

Comment:
Using a plural name for a class feels wrong. I'd suggest to rename it to ThresholdSet or something like that.